### DISCUSSION TITLE
Github Copilot issues

### DISCUSSION THREAD

**Discussion by mehdios36000** [community/community#40117] [GitHub Education]
I was working on a project and I noticed that GitHub copilot ended my subscription for some reason although I have a student pack, What should I do?

**Comment by Sohaib-GO** [comment]
same thing

**Comment by jfullstackdev** [comment]
I'm seeing a lot of these things, you can check this out since one GitHub Staff already responded there

https://github.com/orgs/community/discussions/40037
